# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1341219107 -7200
#      Mon Jul 02 10:51:47 2012 +0200
# Branch gui
# Node ID ed0f820c7ce004b5a8515fc28a86b455026af49a
# Parent  eae0e9f2a8c6c0e2c41851a9816cadd41d219ef7
Changed QImage to QPixmap to allow compilation with QScintilla2 < 2.6.

* file-editor-tab.cc (file_editor_tab()): Changed marker initializer from QImage to QPixmap.

diff --git a/gui/src/m-editor/file-editor-tab.cc b/gui/src/m-editor/file-editor-tab.cc
--- a/gui/src/m-editor/file-editor-tab.cc
+++ b/gui/src/m-editor/file-editor-tab.cc
@@ -29,18 +29,18 @@ file_editor_tab::file_editor_tab(file_ed
   _file_name = "";
   _edit_area = new QsciScintilla (this);
   _edit_area->setLexer (fileEditor->lexer ());
 
   // symbols
   _edit_area->setMarginType (1, QsciScintilla::SymbolMargin);
   _edit_area->setMarginSensitivity (1, true);
   _edit_area->markerDefine (QsciScintilla::RightTriangle, bookmark);
-  _edit_area->markerDefine (QImage (":/actions/icons/redled.png"), breakpoint);
-  _edit_area->markerDefine (QImage (":/actions/icons/arrow_right.png"),
+  _edit_area->markerDefine (QPixmap (":/actions/icons/redled.png"), breakpoint);
+  _edit_area->markerDefine (QPixmap (":/actions/icons/arrow_right.png"),
                             debugger_position);
 
   connect (_edit_area, SIGNAL (marginClicked (int, int, Qt::KeyboardModifiers)),
            this, SLOT (handle_margin_clicked (int, int, Qt::KeyboardModifiers)));
 
   // line numbers
   _edit_area->setMarginsForegroundColor(QColor(96,96,96));
   _edit_area->setMarginsBackgroundColor(QColor(232,232,220));
